nautobot/dcim/api/serializers.py

```diff
@@ -225,6 +225,20 @@ class Meta:
             "computed_fields",
         ]
         opt_in_fields = ["computed_fields"]
+        # Omit the UniqueTogetherValidator that would be automatically added to validate (site, slug). This
+        # prevents slug from being interpreted as a required field.
+        validators = [UniqueTogetherValidator(queryset=RackGroup.objects.all(), fields=("site", "name"))]
```

Comment:
Note likely desirable 2.0 breaking change: require RackGroup `slug` to be globally unique on its own, rather than only unique per site.

Reply:
I will put in a TODO and a ticket.  There are some features we can take advantage of to help here.  `populate_from` can take a list. I can change this to use both `site` and `name`, but doing so will require a change in our tests.  Maybe we make that change in 2.0.